assets/text/faq/faq16-custom-url.md

```diff
@@ -8,14 +8,15 @@ Tap the Parameters to see a list of available parameters that you can use, inclu
 
 You can add your HTTP body, HTTP header, HTTP method and basic authentication credentials in the Custom URL screen.
 
-If your phone goes **offline**, then the app will queue these requests until a data connection becomes available.     
+If your phone goes **offline**, then the app will queue these requests until a data connection becomes available. This behavior can be changed with the 'Discard offline locations' toggle.
 
 If you use a self signed **SSL** certificate, be sure to [validate it first](#customsslcertificates).
 
 The 'Log to custom URL' toggle will send an HTTP request each time the app acquires a new point.  
 
 The 'Allow auto sending' toggle will also enable CSV logging out of necessity, and this feature will then make many requests to the Custom URL, one for each line in the recorded CSV.  It does this at the auto-send interval which is by default 60 minutes.  If you are writing these points to a database for example, it is your responsibility to 'deduplicate' when receiving these requests, for that reason it's a good idea to send a timestamp along.  
 
+The 'Discard offline locations' toggle will disable queueing the log requests while the phone is offline. It will always only keep the request with most recent location. This prevents sending many requests when the phone becomes online after a long period of time. You can enable this toggle if you are interested only in the most recent location. The auto sending feature is not affected by this toggle.
```

Comment:
I see, missed this.  I will test its effect, should be OK.  I added this to the sendByHttp method 

```
            if(!Systems.isNetworkAvailable(AppSettings.getInstance())){
                LOG.debug("Device offline, will not log to Custom URL");
                return;
            }
```

but that would have prevented the latest point (even while offline) from being queued.  Adding it would make Custom URL purely online.

Reply:
Thanks for helping with this. First I thought to keep the latest offline point in queue so the position is sent faster when the phone connects to a network. But I'm totally fine with a purely online solution.
My other idea would be to call the persist() on params in CustomUrlJob only if the condition (!discard offline || has network) is true. This would prevent disk write, however not decrease queue checks and increase code complexity for an edge case.

Reply:
Nice idea, I decided to try out the purely online idea, and that actually greatly simplifies the code.   Just needed to check if the device is online/offline before submitting the job makes it very simple, and I realized no need for tags or cancelling existing jobs.  To help things out a bit I've reduced the number of retries to 3, so after a user's device comes back online, not much delay in sending the latest location (in case any retries were pending).  I'll push it up soon.